cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -221,6 +221,7 @@ func (builder *StakedAccessNodeBuilder) Build() (cmd.Node, error) {
 				builder.rpcMetricsEnabled,
 				builder.apiRatelimits,
 				builder.apiBurstlimits,
+				nil,
```

Comment:
nit: we can drop the `staked_` prefix from the filename of this file

Reply:
Thank you for the review. This is fixed here: https://github.com/onflow/flow-go/pull/2370/commits/eed9711c5ffe742193f3994176df17bfcdbba647